### DISCUSSION TITLE
Missing popup menus

### DISCUSSION THREAD

**Discussion by Octavianbr** [Power-Maverick/DataverseDevTools-VSCode#191] [Q&A] [answered]
Hello!
Just installed the tool in my VSCode.
Created the connection and the tool found the entities and all the rest.
I wanted to create a JS file for one of the entities and use the mapping between local file and the file on the server but a lot of popup menus are missing compared to what I see in the demos. It behaves like it is in R/O mode or even less because I cannot see the content of the JS file.
It must be a setting but I cannot figure out which one.
If you have an idea what I am missing here, it will be greatly appreciated.
Thank you.
<img width="507" alt="VSCode with Dataverse DEvTools" src="https://github.com/user-attachments/assets/52eb38e8-adfe-4df7-a20e-0da577fcb4c2">

**Comment by Power-Maverick** [comment, marked as answer]
To create the JS or TS project or to add new JS or TS file you have to use the explorer. If the project already exists; right-clicking any folder will present you with an option to add JS or TS file. If project is not initialized; right-clicking any empty space will provide you an option to initiate a new project. Hope that helps.
